packages/mobile/src/components/AggressiveWarning/AggressiveWarning.stories.tsx

```diff
@@ -0,0 +1,7 @@
+import { storiesOf } from '@storybook/react-native'
+import React from 'react'
+import AggressiveWarningComponent from './AggressiveWarning.component'
+
+storiesOf('AggressiveWarning', module).add('Default', () => (
```

Comment:
You have to add this story to `.storybook/index.js` otherwise it won't be reachable

Reply:
If you will, you can automate it to be included automatically like in desktop package I believe